src/js/common/schemaform/FormPage.jsx

```diff
@@ -118,31 +136,50 @@ class FormPage extends React.Component {
           uiSchema={uiSchema}
           validate={this.validate}
           showErrorList={false}
-          formData={this.state.formData}
-          widgets={widgets}
-          fields={fields}
+          formData={formData}
+          widgets={reviewMode ? reviewWidgets : widgets}
+          fields={reviewMode ? reviewFields : fields}
           transformErrors={this.transformErrors}>
-        <div className="row form-progress-buttons schemaform-buttons">
-          <div className="small-6 medium-5 columns">
-            <ProgressButton
-                onButtonClick={this.goBack}
-                buttonText="Back"
-                buttonClass="usa-button-outline"
-                beforeText="«"/>
-          </div>
-          <div className="small-6 medium-5 end columns">
-            <ProgressButton
-                submitButton
-                buttonText="Continue"
-                buttonClass="usa-button-primary"
-                afterText="»"/>
-          </div>
-        </div>
+        {reviewPage && !reviewMode &&
+          <ProgressButton
+              submitButton
+              buttonText="Update page"
+              buttonClass="usa-button-primary"/>}
+        {!reviewPage &&
+          <div className="row form-progress-buttons schemaform-buttons">
+            <div className="small-6 medium-5 columns">
+              <ProgressButton
+                  onButtonClick={this.goBack}
+                  buttonText="Back"
+                  buttonClass="usa-button-outline"
+                  beforeText="«"/>
+            </div>
+            <div className="small-6 medium-5 end columns">
+              <ProgressButton
+                  submitButton
+                  buttonText="Continue"
+                  buttonClass="usa-button-primary"
+                  afterText="»"/>
+            </div>
+          </div>}
       </Form>
     );
   }
 }
 
+function mapStateToProps(state, ownProps) {
+  return {
+    form: state.form,
+    route: ownProps.route,
+    reviewMode: ownProps.reviewMode
```

Comment:
I was under the impression that react-redux wouldn't pass through props you passed to the connect wrapped component to the inner component. I'm not sure I have tried that directly, so I could be wrong.